modules/post-processing/lane-detection/modulestack.yaml

```diff
@@ -0,0 +1,54 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This template deploys a Module specific IAM permissions
+
+Parameters:
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "s3:Create*"
+              - "s3:Put*"
+              - "s3:Delete*"
+              - "s3:Get*"
+              - "s3:List*"
+            Effect: Allow
+            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+          - Action:
+              - "secretsmanager:Get*"
+              - "secretsmanager:DescribeSecret"
+              - "secretsmanager:ListSecretVersionIds"
+            Effect: Allow
+            Resource: !Sub "arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:addf*"
```

Comment:
we need to revisit why this secretsmanager section is getting copy/pasted into all new modules. it shouldn't be needed. seed-farmer already adds permissions to the dockerCredentialsSecret to the Module Deployment Role. if that isn't working, we should fix the bug and remove these permissions.